src/main/java/com/google/cloud/spanner/pgadapter/statements/CopyStatement.java

```diff
@@ -242,6 +241,8 @@ private void parseCopyStatement() throws Exception {
       parse(sql, this.options);
     } catch (Exception e) {
       throw new SQLException("Invalid COPY statement syntax: " + e.toString());
+    } catch (TokenMgrError e) {
```

Comment:
nit: you can combine the two catch clauses into one like this:

```
    } catch (Exception | TokenMgrError e) {
      throw new SQLException("Invalid COPY statement syntax: " + e.toString());
    }
```

Reply:
Done.